api/ncf_api_flask_app/views.py

```diff
@@ -12,6 +12,7 @@
 sys.path[0:0] = [NCF_TOOLS_DIR]
 
 default_path = ""
+use_rudder_auth = True
```

Comment:
Not sure about that value. Why Rudder by default in ncf? 
On the other hand, the auth was already set to Rudder by default...